src/components/BookForm.js

```diff
@@ -27,6 +27,7 @@ const BookForm = () => {
 
```

Comment:
**[OPTIONAL]**

- [x] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Thank you👍💯 for the suggestion, I will see to it done